microhapulator/filter.py

```diff
@@ -45,6 +45,12 @@ def filter(self):
             else:
                 self.counts.pairs_failed += 1
 
+    @property
+    def summary(self):
+        header = "R1OnlyFailed\tR2OnlyFailed\tPairsFailed\tPairsRemoved\tPairsKept\tTotalPairs"
+        count_data = f"{self.counts.r1_failed}\t{self.counts.r2_failed}\t{self.counts.pairs_failed}\t{self.counts.total_pairs_filtered}\t{self.counts.pairs_passed}\t{self.counts.total_pairs}"
+        return f"{header}\n{count_data}"
+
```

Comment:
I moved this to the parent class because this is likely the summary we will want for any type of read filtering we do.